uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_hw/src/uwrt_mars_rover_drivetrain_hw_actuator_interface.cpp

```diff
@@ -147,7 +164,9 @@ hardware_interface::return_type UWRTMarsRoverDrivetrainHWActuatorInterface::writ
   RCLCPP_DEBUG(logger_, "Drivetrain Actuator Writing...");
 
   RCLCPP_DEBUG_STREAM(logger_, "Joint Velocity: " << joint_velocity_command_);
-  // TODO: write to CAN
+
+  drivetrain_can_wrapper_.writeToID<double>(joint_velocity_command_, write_address_);
```

Comment:
Is this just a filler for now? What you actually want is something along these lines:

```suggestion
  float vel_msg = <some value>;
  auto can_msg_id = (int)write_address_ << 5 | joint_velocity_command_;
  drivetrain_can_wrapper_.writeToID<float>(vel_msg, can_msg_id);
```